BetaCameras/Xtion2/Xtion2.cs

```diff
@@ -12,13 +12,18 @@ namespace MetriCam2.Cameras
     public class Xtion2 : Camera, IDisposable
     {
         private bool _disposed = false;
-        OpenNICAPI.Device _device = new OpenNICAPI.Device();
-        OpenNICAPI.Stream _colorStream = new OpenNICAPI.Stream();
-        OpenNICAPI.Stream _depthStream = new OpenNICAPI.Stream();
-        OpenNICAPI.Stream _irStream = new OpenNICAPI.Stream();
-        OpenNICAPI.Frame _colorFrame = new OpenNICAPI.Frame();
-        OpenNICAPI.Frame _depthFrame = new OpenNICAPI.Frame();
-        OpenNICAPI.Frame _irFrame = new OpenNICAPI.Frame();
+        OpenNI2CApi.Device _device = new OpenNI2CApi.Device();
+        OpenNI2CApi.Stream _colorStream = new OpenNI2CApi.Stream();
+        OpenNI2CApi.Stream _depthStream = new OpenNI2CApi.Stream();
+        OpenNI2CApi.Stream _irStream = new OpenNI2CApi.Stream();
+        OpenNI2CApi.Frame _colorFrame = new OpenNI2CApi.Frame();
+        OpenNI2CApi.Frame _depthFrame = new OpenNI2CApi.Frame();
+        OpenNI2CApi.Frame _irFrame = new OpenNI2CApi.Frame();
+
+        public override string Name
```

Comment:
It's recommended to override `Vendor` and `Model` instead. `Name` uses those.
But you could just skip that, too, as they have default implementations in the base class.

Reply:
Done